hera_cal/vis_clean.py

```diff
@@ -213,6 +225,69 @@ def apply_calibration(self, input_cal, unapply=False):
         apply_cal.calibrate_in_place(self.data, cal_gains, self.flags, cal_flags,
                                      gain_convention=gain_convention)
 
+    def apply_flags(self, external_flags=None, overwrite_data_flags=False,
+                    flag_zero_times=True, a_priori_flag_yaml=None):
+        """
+        apply external flags.
+
+        Parameters
+        ----------
+        external_flags: str, optional.
```

Comment:
why make this only a string? shouldn't this just be a UVFlag object or a string?

Reply:
Sounds good. Adding UVFlag support.

Reply:
Also clarified the Noneness in the docstring and added a ValueError if they are both None.